snackbar/material/index.md

```diff
@@ -0,0 +1,18 @@
+# Aufgabenmaterial herunterladen
+
+Um das Aufgabenmeterial für [Kiosk](../) herunterzuladen, gib folgenden Befehl in _ein neues Terminal_ in deinem Codespace ein:
+
+```
+wget -O - https://inf-lab.dev/kiosk/material/public/scripts/setup.sh | bash
+```
+
+<details>
+    <summary style="font-weight: bold">Ich kann die Aufgabe schon alleine lösen, bin ich jetzt vergebens hier?</summary>
+    Nein! Ihr könnt gerne aufpassen, dass ich keinen Fehler mache und ggf. euren eigenen Code verbessern.
+
+</details>
+
+<details>
+    <summary style="font-weight: bold">Was macht der Befehl überhaupt?</summary>
+    Dieser Befehl lädt Code herunter und führt diesen direkt aus, sodass für euch komfortabel ein weiteres ZIP-Archiv heruntergeladen und gleich entpackt wird. All das wird anschließend wieder aufgeräumt.
+</details>
```

Comment:
This is probably not something we should discuss here, but rather over at [page](https://github.com/inf-lab-dev/page):

As this code segment probably gets reused a lot it should be extracted into a component. If you agree, please create an issue in the [page](https://github.com/inf-lab-dev/page) repository.